OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitDiscussionFragment.java

```diff
@@ -36,6 +47,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
     public void onViewCreated(View view, Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
         if (savedInstanceState == null) {
+            courseApi.markBlocksCompletion(unit.getCourseId(), new String[]{unit.getId()}).enqueue(new Callback<JSONObject>() {
+                @Override
+                protected void onResponse(@NonNull JSONObject responseBody) {
+                    // Nothing to do here
+                }
+            });
```

Comment:
I think `markBlockCompletion` should be in `CourseDiscussionPostsThreadFragment` after getting the discussion response. @miankhalid thoughts?

Reply:
for that, we have to pass the related values to the next fragment

Reply:
okay, we can ignore it for now.